idaes/apps/grid_integration/pricetaker/clustering.py

```diff
@@ -229,24 +306,13 @@ def get_optimal_num_clusters(
     return n_clusters
 
 
-def locate_elbow(x: list, y: list):
```

Comment:
Why was this removed?

Reply:
The get_optimal_num_clusters method above handles this. This was really just a placeholder method for when I added the literature-based implementation